internal/patch/patch.go

```diff
@@ -44,6 +44,19 @@ func NewValidator(patchReq string, s schema.Schema, extensions ...schema.Schema)
 		return OperationValidator{}, err
 	}
 
+	switch v := operation.Value.(type) {
+	// Okta also send the ID on PATCH requests.
+	// See: internal/idp_test/testdata/okta/update_group_name.json
+	// https://developer.okta.com/docs/reference/scim/scim-20/#update-a-specific-group-name
+	case map[string]interface{}:
+		for k := range v {
+			if strings.ToLower(k) == "id" {
+				delete(v, k)
+				break
```

Comment:
Should we consider maps with multiple keys that case insensitively equal `"id"`?

Reply:
We should. I'll fix this.